forge-gui/res/cardsfolder/s/stomping_slabs.txt

```diff
@@ -1,7 +1,7 @@
 Name:Stomping Slabs
 ManaCost:2 R
 Types:Sorcery
-A:SP$ Dig | Cost$ 2 R | DigNum$ 7 | Reveal$ True | RememberRevealed$ True | DestinationZone$ Library | LibraryPosition$ -1 | LibraryPosition2$ -1 | SubAbility$ DBDamage | SpellDescription$ Reveal the top seven cards of your library, then put those cards on the bottom of your library in any order. If a card named CARDNAME was revealed this way, CARDNAME deals 7 damage to any target.
+A:SP$ Dig | DigNum$ 7 | Reveal$ True | RememberRevealed$ True | ChangeNum$ 0 | DestinationZone2$ Library | SubAbility$ DBDamage | SpellDescription$ Reveal the top seven cards of your library, then put those cards on the bottom of your library in any order. If a card named CARDNAME was revealed this way, CARDNAME deals 7 damage to any target.
```

Comment:
Hmm, should desc be `a card named CARDNAME` or `a card named Stomping Slabs`?

Not sure of how this works in this case

Reply:
should make the most difference on permanents but still...